cordova-plugin-appcenter-push/src/android/AppCenterPushPlugin.java

```diff
@@ -23,12 +22,6 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
                 cordova.getActivity().getApplication(),
                 webView.getPreferences());
 
-        // For some reason Cordova reads SENDER_ID preference as double. 
-        // Because of this Pushes does not work properly, 
-        // as workaround SENDER_ID value should be wrapped by single quotes.
-        String senderId = webView.getPreferences().getString(SENDER_ID, "").replace("'", "");
-        Push.setSenderId(senderId);
```

Comment:
Since we apply plugin automatically they are forced to put the google-services.json file. Thus this is useless to maintain senderId code. Since we are not 1.0.0 we can do breaking changes.